media/redesign/stylus/newsletter.styl

```diff
@@ -1,17 +1,3 @@
 @require 'includes/mixins';
 @require 'includes/vars';
-
-#apps-newsletter-subscribe {
-    #newsletter-toggle {
-        @extend $checkbox-label-container;
-    }
-
-    .newsletter-setting {
-        display: none; /* Prevents flash of content */
-        padding-left: $grid-spacing;
-
-        .no-js & {
-            display: block;
-        }
-    }
-}
+@require 'components/newsletter';
```

Comment:
The newsletter.css file is compiled and attached to at least three pages on its own instead of being required, at the moment. Not sure we want to weigh down profile.css with it but it's possible we could move it into submission.css. I've created Bug 1120698 to review this separately.